cookbook/deployment/customizing_resources.py

```diff
@@ -48,23 +48,21 @@ def count_unique_numbers(x: typing.List[int]) -> int:
 
 
 # %%
-# Now lets create a dummy task that squares the number
+# Now lets create a dummy task that determines the square of a number.
```

Comment:
```suggestion
# Let's create a dummy task that determines the square of a number.
```

Reply:
Will work on this in a separate PR. I just saw a spelling error and correct it, but will work on the entire file in another PR @samhita-alla